server/cmd/start.go

```diff
@@ -96,3 +103,21 @@ func Namespace(ns string) gin.HandlerFunc {
 		c.Next()
 	}
 }
+
+func setBaseHRef(filename string, baseHRef string) error {
+	if baseHRef == "/" {
+		return nil
+	}
+
+	file, err := os.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	prevHRef := `<base href="/">`
+	newHRef := fmt.Sprintf(`<base href="%s">`, baseHRef)
+	file = bytes.Replace(file, []byte(prevHRef), []byte(newHRef), -1)
+
+	err = os.WriteFile(filename, file, 0666)
```

Comment:
This is hacky, and it won't work due to permission issue right?

Reply:
I think we should change the line based on the configuration.
```
router.Use(static.Serve("/", static.LocalFile("./ui/build", true)))
```
as well as the line 31 in routes.go.